testing/kfctl/kf_is_ready_test.py

```diff
@@ -145,6 +145,34 @@ def test_kf_is_ready(namespace, use_basic_auth, use_istio, app_path):
     logging.info("Verifying that deployment %s started...", deployment_name)
     util.wait_for_deployment(api_client, knative_namespace, deployment_name, 10)
 
+  if platform == "gcp":
+    # check secret
+    util.wait_for_secret(api_client, namespace, "user-gcp-sa")
```

Comment:
Why do we need to "wait_for_secret". The secret should already exist right? We don't need to wait for the secret to be ready.

Reply:
A better name is "check_secret"